config/initializers/config.rb

```diff
@@ -0,0 +1,77 @@
+Config.setup do |config|
+  # Name of the constant exposing loaded settings
+  config.const_name = 'Settings'
```

Comment:
```suggestion
  config.const_name = "Settings"
```
Các file ruby và yml, e chuyển qua dùng nháy đôi hết nha
Tiện nhất là setup rubucop để tự động fix các lỗi nháy đôi trong file .rb